src/nxt_sockaddr.c

```diff
@@ -734,6 +734,11 @@ nxt_sockaddr_inet_parse(nxt_mp_t *mp, nxt_str_t *addr)
 
     inaddr = INADDR_ANY;
 
+    if (length == 0) {
+        nxt_thread_log_error(NXT_LOG_ERR, "invalid size \"%V\"", length);
```

Comment:
The logging could be like this:
```
(..., "invalid address \"%V\"", addr);
```

Reply:
The length check should really come before `inaddr = INADDR_ANY;`... but lets see if this is really the right place for this...

Reply:
Makes sense.